### PR TITLE
feat: Banner Styling Fixes

### PR DESCRIPTION
**Pull request by Rajani169**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -41,6 +47,9 @@ export default () => {
   const classes = useStyles()
   const theme = useTheme()
   const matches = useMediaQuery('(min-width:640px)')
+  const active = useMediaQuery('(min-width:960px)')
+  const FontLocation = active ? { fontSize: 30 } : null
+  const FontCalendar = active ? { fontSize: 30 } : null
```
Comment (amritsrivastava) [inline src/pages/index.js]: Why you need this @Rajani169 when you defined the styling above in the classes, implement the styling for both venue and date using classes only, implement the media queries in classes.

File: src/pages/index.js [hunk 2]
```diff
@@ -98,11 +107,11 @@ export default () => {
           </Typography>
         </div>
         <div className='my-10'>
-          <Typography className='text-white' variant='h6'>
+          <Typography className='text-white' style={FontLocation}>
```
Comment (amritsrivastava) [inline src/pages/index.js]: don't use style tag use classes with `makeStyles` and use `classnames` for implementing this.

File: src/pages/index.js [hunk 3]
```diff
@@ -98,11 +107,11 @@ export default () => {
...
             <LocationIcon color='primary' /> Jaipur Engineering College and
             Research Center, Jaipur
           </Typography>
-          <Typography className='text-white' variant='h6'>
+          <Typography className='text-white' style={FontCalendar}>
```
Comment (amritsrivastava) [inline src/pages/index.js]: Same here use classes

**Review by amritsrivastava — CHANGES_REQUESTED** [review]

File: src/pages/index.js [hunk 4]
```diff
@@ -26,13 +26,19 @@ const useStyles = makeStyles(theme => ({
...
   },
   diplomacyTextProperty: {
     color: '#fff',
     fontFamily: "'Rubik' , sans-serif",
     fontWeight: 'bold',
     fontSize: '24px',
     lineHeight: '28px',
+    [theme.breakpoints.up('md')]: {
+      fontSize: 35,
+    },
```
Comment (amritsrivastava) [inline src/pages/index.js:42]: also increase the `lineHeight` so that diplomacy and jecrc MUN will get a space between them

**Review by amritsrivastava — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 1 above)
Comment (Rajani169) [reply src/pages/index.js]: Done!!

File: src/pages/index.js (hunk 2 above)
Comment (Rajani169) [reply src/pages/index.js]: Okay

File: src/pages/index.js (hunk 3 above)
Comment (Rajani169) [reply src/pages/index.js]: Okay. done.

File: src/pages/index.js (hunk 4 above)
Comment (Rajani169) [reply src/pages/index.js:42]: Done

File: src/pages/index.js [hunk 5]
```diff
@@ -26,13 +27,29 @@ const useStyles = makeStyles(theme => ({
...
+  FontCalendar: {
+    [theme.breakpoints.up('md')]: {
+      fontSize: 30,
+    },
+  },
+  FontLocation: {
+    [theme.breakpoints.up('md')]: {
+      fontSize: 30,
+    },
   },
```
Comment (amritsrivastava) [inline src/pages/index.js]: Do you really need two classes with same properties for two different typography elements ??
Can't you apply the single class to multiple elements ??

**Review by amritsrivastava — DISMISSED** [review]

File: src/pages/index.js (hunk 5 above)
Comment (Rajani169) [reply src/pages/index.js]: Sorry My mistake

**Review by amritsrivastava — APPROVED** [review]

**Pull request merged**